src/core/server/plugins/plugins_service.test.ts

```diff
@@ -143,19 +143,18 @@ describe('PluginsService', () => {
       `);
     });
 
-    it('throws if plugin required Kibana version is incompatible with the current version', async () => {
+    it('does not throw if plugin required Kibana version is incompatible with the current version', async () => {
```

Comment:
Is there a discussion about changing this behavior? I was sure we want to preserve it.